faq/templates/faq/inclusion_tags/faq_detail.html

```diff
@@ -2,12 +2,10 @@
 {% load ultracache_tags %}
 
 {% block content %}
-    {% ultracache 3600 "object-detail-inclusion" object.id object.modified request.page %}
-        {% with object.content_pages as content_pages %}
-            {% autopaginate content_pages 1 %}
-            <div class="content">
-                <a name="jmbo-faq"></a>
-                {{ content_pages.0|safe }}
+    {% ultracache 3600 "object-detail" object.id object.modified %}
```

Comment:
Call the key `object-detail-inclusion` because `object-detail` is used by another template in Jmbo itself.